src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -1346,14 +1346,14 @@ export class SymbolInformation {
 	location!: Location;
 	kind: SymbolKind;
 	tags?: SymbolTag[];
-	containerName: string | undefined;
+	containerName: string;
```

Comment:
@jrieken Without this change, I get errors that `extHostTypes.SymbolInformation` is not assignable to `vscode.SymbolInformation` (they are assignable in the opposite direction)

This seemed wrong but let me know if I should add a type assertion instead